openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -289,7 +289,7 @@ def _register_parameter_handler_classes(self, parameter_handler_classes):
             #tagname = subclass._TAGNAME
             tagname = parameter_handler_class._TAGNAME
             if tagname is not None:
-                if tagname in self._parameter_handler_classes.keys():
+                if tagname in self._parameter_handler_classes:
```

Comment:
I usually remove them because this
```
if key in dict
```
has O(1) computational complexity since it just checks the key hash, while I _think_ that
```
if key in dict.keys()
```
ends up generating a list and iterate over all the keys so it is O(n), unless Python ends up doing some sort of optimization at interpretation time. I agree that for small dictionaries the speedup is irrelevant, but if we want to stick to a single convention, we should probably use the fastest syntax.

Reply:
I agree. What you said is the convention we should use from now on.